apps/web/src/assets/css/web.scss

```
@@ -1,11 +1,28 @@
+// MOJ Frontend selectively @use'd before govuk-frontend's index: the MOJ vendor
+// layer configures the shared govuk "base" module with `with (...)`, which Sass
+// only allows before that module is first loaded. govuk-frontend's index then
+// @forwards the same (already configured) base module without `with`, which is
+// permitted. Only the filter component and its layout object are pulled in to
+// avoid re-asserting GDS defaults over the rebrand overrides further down.
+@use "@ministryofjustice/frontend/moj/components/filter/filter";
+@use "@ministryofjustice/frontend/moj/objects/filter-layout";
+// The filter toggle (FilterToggleButton) shows/hides the panel by toggling
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)